docs/omnipaxos/leader_election.md

```diff
@@ -10,3 +10,29 @@ omni_paxos.election_timeout();
 If a leader has failed, it will be detected in one election timeout and a new leader will be elected in the next timeout (if possible).
 
 > **Note:** The `leader_priority` field in `OmniPaxosConfig` allows user to give desired servers a higher priority to get elected upon a leader change.
+
+## Flexible Quorums
+OmniPaxos also offers support for flexible quorums. Normally a node needs to consult a majority of nodes to become a leader and then as a leader consult a majority of nodes to append to the log. Flexible quorums allow for a trade off between the number of nodes that need to be consulted in each of these two scenarios.
+
+OmniPaxos can be configured to use a flexible quorum as follows:
+```rust,edition2018,no_run,noplaypen
+use omnipaxos::{OmniPaxosConfig, util::FlexibleQuroum};
+
+let flex_quorum = FlexibleQuorum {
+    read_quorum_size: 5,
+    write_quorum_size: 3,
+};
+
+let config = OmniPaxosConfig {
+    configuration_id: 1,
+    pid: 1,
+    peers: vec![2, 3, 4, 5, 6, 7],
+    flexible_quorum: Some(flex_quorum),
+    ..Default::deafult()
+};
+```
+In OmniPaxos, becoming a leader involves reading from the log and so `read_quorum_size` is the number of nodes to consult in order to become a leader. The `write_quorum_size` is the number of nodes to consult in order to append to the log.
+
+Appending to the log is a much more common occurance than electing a leader so reducing the `write_quorum_size` can reduce the latency and sometimes even increase throughput. Note, however, that reducing the `write_quorum_size` also decreases the fault tolerance of the cluster to `write_quorum_size - 1`.
```

Comment:
```suggestion
Appending to the log is a much more common occurrence than electing a leader, so reducing the `write_quorum_size` can be beneficial. Note, however, that reducing the `write_quorum_size` also requires a larger `read_quorum_size`, which may reduce fault tolerance.
```

Reply:
I saw that you wrote something similar right afterwards. Then I would consider removing the last sentence here.